site/en/docs/devtools/elements/badges/index.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Badges reference"
+date: 2022-08-02
+#updated: YYYY-MM-DD
+description: "Toggle various overlays and speed up DOM tree navigation with badges."
+authors:
+  - sofiayem
+tags:
+  - dom
+  - css
+  - html
+---
+
+Toggle various overlays and speed up DOM tree navigation with this comprehensive reference of badges in the **Elements** panel.
+
+## Show or hide badges
+
+To show or hide badges:
+
+1. [Open DevTools](/docs/devtools/open/#elements).
+1. Right-click an element in the DOM tree and select **Badge settings...**.
+   {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/xpRsrsvl9BxxANM4Xuiy.png", alt="Badge settings.", width="800", height="337" %}
+1. Select or clear checkboxes next to the desired badges.
+
+The **Elements** panel shows the selected badges next to the appropriate elements in the DOM tree. The next sections explain every badge.
+
+## Grid
+
+An HTML element is a [grid container](https://developer.mozilla.org/docs/Web/CSS/CSS_Grid_Layout) if its `display` CSS property is set to `grid` or `inline-grid`. Such elements have `grid` badges next to them that toggle the corresponding overlays.
+
+Toggle the overlay on the following preview:
+
+{% Codepen {
+ user: 'sofiayem',
+ id: 'YzarzKV',
+ height: 270,
+ allow: ['geolocation']
+} %}
+
+1. [Inspect the element][1] in the preview.
+1. In the DOM tree, click the `grid` badge next to the element and observe the overlay.
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/WMZrfd1AVdMjkgCDqSK8.png", alt="Grid overlay.", width="800", height="674" %}
+
+The overlay shows columns, rows, their numbers, and gaps.
+
+To learn how to debug grid layout, see [Inspect CSS grid](/docs/devtools/css/grid/).
+
+## Flex
+
+An HTML element is a [flex container](https://developer.mozilla.org/docs/Web/CSS/CSS_Flexible_Box_Layout) if its `display` CSS property is set to `flex` or `inline-flex`. Such elements have `flex` badges next to them that toggle the corresponding overlays.
+
+Toggle the overlay on the following preview:
+
+{% Codepen {
+  user: 'sofiayem',
+  id: 'XWEeWdb',
+  height: 200,
+  allow: ['geolocation']
+} %}
+
+1. [Inspect the element][1] in the preview.
+1. In the DOM tree, click the `flex` badge next to the element and observe the overlay.
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/MmkJORNp3sOKHHK1CI4h.png", alt="Flex overlay.", width="800", height="646" %}
+
+The overlay shows child element positions. 
+
+To learn how to debug flex layouts, see [Inspect and debug CSS flexbox](/docs/devtools/css/flexbox/).
+
+## Ad
+
+DevTools can detect most ad frames. Such frames have `ad` badges next to them. 
```

Comment:
expand explanation on how we know it's an ad frame. Tag you in a conversation and resource.